### PR TITLE
Display payment methods in the deployment flow

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- see me, unless you know what you're doing in which case go ahead and see if it works...

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6965/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6979/

File: jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js [hunk 1]
```diff
@@ -900,6 +947,12 @@ YUI.add('deployment-flow', function() {
       if (this.state.loadingTerms) {
         return false;
       }
+      // Can't deply if there is no user.
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js]: deploy

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6996/

**Comment by huwshimi** [comment]
Thanks @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**